src/main/groovy/com/github/hauner/openapi/spring/writer/DataTypeWriter.groovy

```diff
@@ -47,6 +59,9 @@ class DataTypeWriter {
             def javaPropertyName = Identifier.toCamelCase (it)
             def propDataType = dataType.getObjectProperty (it)
             target.write ("    @JsonProperty(\"$it\")\n")
+            if(apiOptions.beanValidation){
+                target.write (beanValidationWriter.createAnnotations (propDataType))
+            }
```

Comment:
Guess its worth extracting a method here

            target.write (getProperty (javaPropertyName, propDataType))

so it looks similar to the getter & setter (the `each` below)